src/instruments/src/EFB/Dispatch/Pages/PayloadPageLayout.tsx

```diff
@@ -0,0 +1,361 @@
+        },
+        aftCont: {
+            name: 'AFT CONTAINER',
+            weight: 2426,
+            currentWeight: 0,
+            stationIndex: 7 + 1,
+            position: -15.96,
+        },
+        aftBag: {
+            name: 'AFT BAGGAGE',
+            weight: 2110,
+            currentWeight: 0,
+            stationIndex: 8 + 1,
+            position: -27.10,
+        },
+        aftBulk: {
+            name: 'COMP 5 - AFT BULK/LOOSE',
+            weight: 1497,
+            currentWeight: 0,
+            stationIndex: 9 + 1,
+            position: -37.35,
+        },
+    };
+
+    const dispatch = useDispatch();
+
+    const payload = useSelector((state: RootState) => state.payload);
+
+    const zfwcg = getZfwcg().toFixed(2);
+
+    // const totalPax = Object.values(payload).map((station) => station.pax).reduce((acc, cur) => acc + cur);
+    const totalPaxTarget = Object.values(payload).map((station) => station.paxTarget).reduce((acc, cur) => acc + cur);
+
+    const currentUnit = () => 'KG';
+    const convertUnit = () => 1;
+
+    function setPax(numberOfPax) {
+
+    }
+
+    function getPayload() {
+        return 15000;
+    }
+
+    function setSeatColors() {
+        function paintSeat(id: string, clear: boolean = false) {
+            const seatGroupElement = document.getElementById(id);
+            if (seatGroupElement === null) {
+                return null;
+            }
+            const seatPolygon = seatGroupElement.children[0] as HTMLElement;
+
+            if (clear) {
+                seatPolygon.style.fill = 'inherit';
+            } else {
+                seatPolygon.style.fill = '#00AFB7';
+            }
+            return null;
+        }
+
+        Object.values(payload).forEach((station) => {
+            const [min, max] = station.seatsRange;
+            for (let seatNumber = min; seatNumber <= max; seatNumber++) {
+                paintSeat(seatNumber.toString(), true);
+            }
+        });
+
+        Object.values(payload).forEach((station) => {
+            const [min] = station.seatsRange;
+            for (let seatNumber = min; seatNumber < min + station.pax; seatNumber++) {
+                paintSeat(seatNumber.toString());
+            }
+        });
+    }
+
+    function airplaneCanRefuel() {
+        return true;
+    }
+
+    function totalCurrentPax() {
+        return 100;
+    }
+
+    function formatBoardingStatusClass(baseClass:string, text:boolean) {
+        return `${baseClass} disabled-text`;
+    }
+
+    function formatBoardingStatusLabel() {
+        return '(Unavailable)';
+    }
+
+    function toggleBoardingState() {
+
+    }
+
+    function calculateEta() {
+
+    }
+
+    /**
+     * Calculate %MAC ZWFCG of all stations
+     */
+    function getZfwcg() {
+        const currentPaxWeight = PAX_WEIGHT;
+
+        const leMacZ = -5.233333; // Value from Debug Weight
+        const macSize = 14.0623; // Value from Debug Aircraft Sim Tunning
+
```

Comment:
`const leMacZ = -5.233333; // Value from Debug Weight
        const macSize = 14.0623; // Value from Debug Aircraft Sim Tunning`
These are the old values from the pre-SU4 flight model.  The new values are:
leMacZ = -5.386
macSize = 13.454
Please update.
Would be nice if this PR would go forward!